docs/content/en/4.providers/cloudflare.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Cloudflare Provider
+description: 'Nuxt Image has first class integration with Cloudflare'
+navigation:
+  title: Cloudflare
+---
+
+Integration between [Cloudflare](https://developers.cloudflare.com/images/) and the image module.
```

Comment:
Would you please mention availability only for Business and Enterprise Customers? Normally Cloudflare [pages](https://pages.cloudflare.com/) users can already benefit from build-time image optimization.